pkg/webhook/cluster/validation/validation.go

```diff
@@ -37,9 +37,10 @@ import (
 )
 
 var (
-	supportedCNIPlugins        = sets.NewString(kubermaticv1.CNIPluginTypeCanal.String())
+	supportedCNIPlugins        = sets.NewString(kubermaticv1.CNIPluginTypeCanal.String(), kubermaticv1.CNIPluginTypeCilium.String())
 	supportedCNIPluginVersions = map[kubermaticv1.CNIPluginType]sets.String{
-		kubermaticv1.CNIPluginTypeCanal: sets.NewString("v3.8", "v3.19"),
+		kubermaticv1.CNIPluginTypeCanal:  sets.NewString("v3.8", "v3.19"),
+		kubermaticv1.CNIPluginTypeCilium: sets.NewString("1.10.4"),
```

Comment:
shouldn't it rather be `v1.10.4` to match with the tag? https://github.com/cilium/cilium/releases/tag/v1.10.4

Reply:
good catch!